wikimedia/wikipedia-ios
# Add `TalkPageViewModel`

**Phabricator:** https://phabricator.wikimedia.org/tag/ios-app-feature-talkpages

### Notes
Adds `TalkPageViewModel`, updates `TalkPageViewController` to use it, and updates call sites to use this method of instantiating the new talk page view controller.

### Test Steps
1. On a staging build, confirm the new talk page controller displays via the same paths as before (Settings > Account > Your talk Page, etc.)
